.github/workflows/fly.yaml

```diff
@@ -0,0 +1,58 @@
+name: Fly Preview
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    env:
+      FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+      FLY_ORGANIZATION_SLUG: supabase-dev
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+      - id: deployment
+        uses: sweatybridge/fly-preview@main
+        env:
+          # Specifies the base project to branch from
+          PROJECT_REF: qiao-all-in-one
+          DB_ONLY: true
+      - run: |
+          body="Status: [Deploying](https://fly.io/apps/${{ github.head_ref }}) ($(date))"
```

Comment:
I think we need to include more than `github.head_ref` here - my App was on `seed.fly.io`, but there could be multiple branches called `seed` in various repos

I think that it would require us to preprend the org and repo name. I think it would be:

`${{ github.repository_owner }}.${{ github.event.repository.name }}.${{ github.head_ref }}`

Reply:
Yup, the hostname is now derived from the output of fly deployment action so it also stays in sync with cli.